gnomad/gnomadv4_extract_test.py

```diff
@@ -0,0 +1,50 @@
+# LT 23/08/2022
+
+## Extract a small subset of 10 transcripts from gnomADv4
+## annotate with frequencies
+## export the rows as a hail.Table
+
+
+# target transcripts for panel of initial 10 genes selected by the structural constraint project
+# TODO should be in an external file
+
+TRANSCRIPTS = [
+    "ENST00000366684", # ACTA1
+    "ENST00000371953",  # PTEN
+    "ENST00000298910", # LRRK2
+    "ENST00000320868", # HBA1
+    "ENST00000359125", # KCNQ2
+    "ENST00000393087", # SERPINA1
+    "ENST00000281708",  # FBXW7
+    "ENST00000640769", # RTTN
+    "ENST00000270139", # IFNAR1
+    "ENST00000262030" # ATP5F1B
+]
+
+# connect to hail, using appropriate requester_pays setup
+import hail as hl
+hl.init(default_reference='GRCh38',
+        spark_conf = {
+            'spark.hadoop.fs.gs.requester.pays.mode': 'CUSTOM',
+            'spark.hadoop.fs.gs.requester.pays.buckets': 'gnomad',
+            'spark.hadoop.fs.gs.requester.pays.project.id': 'constraint-232598',
+        })
+
+# gnomAD v4 exomes VDS
+big_vds = hl.vds.read_vds('gs://gnomad/v4.0/raw/exomes/gnomad_v4.0.vds')
+
+# find genomics coordinates of target transcripts and filter vds
+target_intervals = hl.experimental.get_gene_intervals(transcript_ids=TRANSCRIPTS, reference_genome='GRCh38')
```

Comment:
Do we want to use a specific GENCODE version here?

Reply:
nope, I prefer to use the gnomAD standard for GRCh38

Reply:
Not sure what you mean by gnomAD standard, but the GENCODE version differs for each gnomAD version: https://gnomad.broadinstitute.org/help/what-version-of-gencode-was-used-to-annotate-variants

The Hail default GENCODE version is 29, which is fairly old: https://hail.is/docs/0.2/experimental/index.html#hail.experimental.get_gene_intervals

Reply:
Yes, I am aware of that and what to stick to the default version of GENCODE for GRCh38 V29

Reply:
Why though? That's neither compatible with v2, nor v3.1, nor the VEP version which we'd presumably would be running on gnomAD v4, right?

Reply:
Change to GENCODE v39 for full compatibility with Alphafold2